scripts/vcs_wrapper

```diff
@@ -35,4 +40,5 @@ done
 
 IFS="$OIFS"
 
-vcs "${file_args[@]}" "${include_args[@]}" "${library_args[@]}" "${static_lib_args[@]}" "${pli_tab_args[@]}" "$@"
+vcs "${file_args[@]}" "${include_args[@]}" "${filelistargs[@]}" "${library_args[@]}" "${static_lib_args[@]}" "${pli_tab_args[@]}" "$@"
+
```

Comment:
Could you remove this extra blank line you added to the file? In general, we should avoid adding superfluous whitespace in PRs.